bitnami/solr/templates/_helpers.tpl

```diff
@@ -0,0 +1,92 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "solr.name" -}}
+{{- include "common.names.name" . -}}
+{{- end -}}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
+*/}}
+{{- define "solr.fullname" -}}
+{{- include "common.names.fullname" . -}}
+{{- end -}}
+
+{{/*
+Define the name of the solr exporter
+*/}}
+{{- define "solr.exporter-name" -}}
+{{- printf "%s-%s" (include "solr.fullname" .) "exporter" | trunc 63 | trimSuffix "-" -}}
+{{- end -}}
+
+{{/*
+The name of the zookeeper headless service
+*/}}
+{{- define "solr.zookeeper.service" -}}
```

Comment:
This is not being used

Reply:
that's true I changed it by `solr.zookeeper.host`. Deleting it